.github/workflows/main.yml

```diff
@@ -26,16 +33,20 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
-    - name: Lint with flake8
+    - name: Lint with flake8 and black
```

Comment:
If you split black and flake8 into separate "steps" you would get a couple benefits:

 - the steps can run in parallel (probably not a big deal since they're both fast)
 - the GH workflow interface will show you which of flake8 or black formatting failed

Reply:
Thanks for pointing this out. I will fix the commit I pushed and separate the two into different steps.